libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -96,6 +104,27 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     this.items[productIndex].status = ItemsFilters.NotPicked;
 
     this.items = [...this.items];
+
+    this.tabRef.value?.dispatchEvent(
+      new CustomEvent('click', {
+        composed: true,
+        bubbles: true,
+      })
+    );
+
+    setTimeout(() => {
+      window.scrollTo({
+        left: 0,
+        top: this.productCardRef.value?.offsetTop,
+        behavior: 'smooth',
+      });
+
+      (
+        this.productCardRef.value?.renderRoot.querySelector(
+          'oryx-cart-quantity-input'
+        ) as HTMLElement
+      )?.focus();
```

Comment:
Can we delegate focusing to the product card component instead of reaching into it's shadowRoot?
Maybe the card can do the focus automatically when it's switched to edit mode?

Reply:
Sure that's fine, what I mean is to add facade method to the product card component that will focus the input internally and here you would just call that method instead of reaching into the component internals.